Packs/AWS-SecurityHub/Integrations/AWS_SecurityHub/AWS_SecurityHub.yml

```diff
@@ -3332,15 +3385,41 @@ script:
     description: Deprecated, use aws-securityhub-batch-update-findings instead.
     execution: false
     name: aws-securityhub-update-findings
-  dockerimage: demisto/boto3py3:1.0.0.41082
+  - name: get-remote-data
+    deprecated: false
+    description: Get remote data from a remote incident. This method does not update
+      the current incident, and should be used for debugging purposes only.
+    execution: false
+    arguments:
+    - name: id
+      default: false
+      description: The remote incident ID.
+      isArray: false
+      required: true
+      secret: false
+    - name: lastUpdate
+      default: false
+      defaultValue: '0'
+      description: The UTC timestamp in seconds of the last update. The incident is only updated if it was modified after the last update time.
```

Comment:
```suggestion
      description: The UTC timestamp in seconds since the last update. The incident is only updated if it was modified after the last update time.
```

Reply:
What about this?